client/components/add-new-site/content/a4a/index.tsx

```diff
@@ -3,21 +3,14 @@ import AddNewSitesA4AModals from 'calypso/components/add-new-site/modals/a4a';
 import AddNewSitePopover from 'calypso/components/add-new-site/popover';
 import type { AddNewSiteContentProps } from 'calypso/components/add-new-site/types';
 
-const AddNewSiteA4A = ( {
-	isMenuVisible,
-	popoverMenuContext,
-	setMenuVisible,
-	toggleMenu,
-}: AddNewSiteContentProps ) => {
+const AddNewSiteA4A = ( { isMenuVisible, setMenuVisible, toggleMenu }: AddNewSiteContentProps ) => {
 	return (
 		<>
-			<AddNewSitePopover
-				isMenuVisible={ isMenuVisible }
-				toggleMenu={ toggleMenu }
-				popoverMenuContext={ popoverMenuContext }
-			>
-				<AddNewSiteA4AMenuItems setMenuVisible={ setMenuVisible } />
-			</AddNewSitePopover>
+			<div>
```

Comment:
This extra div is required to ensure the popover calculates the correct size using only the button instead of the entire container.